reactive/kubernetes_master.py

```diff
@@ -1853,9 +1867,14 @@ def token_generator(length=32):
     return token
 
 
-@retry(times=3, delay_secs=10)
+@retry(times=3, delay_secs=1)
 def get_pods(namespace='default'):
-    cmd = ['kubectl', 'get', 'po', '-n', namespace, '-o', 'json']
+    cmd = [
+        'kubectl', 'get', 'po',
+        '-n', namespace,
+        '-o', 'json',
+        '--request-timeout', '10s'
```

Comment:
1) 10s seems extreme. Do we need to be that conservative?
2) Should this use the kubectl function in kubernetes_common?

Reply:
1. Can you clarify what you're wanting here? Is 10s too long or too short?

The default is 1m. I lowered it to 10s to reduce the amount of time kubernetes-master waits for futile kubectl calls during the migration. The apiserver can't reach etcd until kubernetes-master processes the `etcd-relation-changed` hook, and this was holding it up.

2. Sure, may as well.

Reply:
My thought is that even 1 second seems way overkill in a local network. If this will connect to an apiserver that comes up within 10 seconds I could see keeping it this long. One minute was extreme and 10 seconds seems too long. I'm not strongly opposed to it, I just thought I would comment.

Reply:
Thanks. I'd like to be cautious and err on the side of leaving it too long. Even with 10s here, most of kubernetes-master's time is spent elsewhere during the transition.